### PR TITLE
2017 community survey post

### PR DESCRIPTION
**Pull request by erickt**
Add a post announcing the 2017 community survey.

### CODE REVIEW

File: _posts/2017-05-03-survey.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+[here][2016 survey].
+
+Please help us spread the word by sharing the above link on your social network
+feeds, at meetups, around your office and in other communities.
+
+If you have any questions, please see our [frequently asked questions] or email
+the Rust Community team at [community-team@rust-lang.org].
+
+Finally, we wanted to thank everyone who helped develop, polish, and test the
+survey! Once it closes, we will summarize and visualize the results here on
```
Comment (Mark-Simulacrum) [inline _posts/2017-05-03-survey.md]: This sentence appears to trail off.

**Comment by aturon** [comment]
Looks great! (Other than the nit from @Mark-Simulacrum)

**Comment by aturon** [comment]
cc @rust-lang/core, this is the survey blog post.

File: _posts/2017-05-03-survey.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Announcing the 2017 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the second year of Rust"
+---
+
+Rust's second birthday is a little less than 2 weeks away (May 15th, 2017), so
```
Comment (steveklabnik) [inline _posts/2017-05-03-survey.md]: two (numbers ten or less should be spelled out)

not a big deal though

**Pull request merged**